doc/onboarding.md

```diff
@@ -204,6 +204,7 @@ Landing a PR
 * Optional: Force push the amended commit to the branch you used to open the pull request. If your branch is called `bugfix`, then the command would be `git push --force-with-lease origin master:bugfix`. When the pull request is closed, this will cause the pull request to show the purple merged status rather than the red closed status that is usually used for pull requests that weren't merged. Only do this when landing your own contributions.
 * `git push upstream master`
     * Close the pull request with a "Landed in `<commit hash>`" comment.
+* Additionally: The [core-validate-commit](https://github.com/evanlucas/core-validate-commit) command validates the commit message for a particular commit in node core.
```

Comment:
Shall I write `Optional:` or `Suggested:` before the bullet point to point to the fact that the core-validate-commit is not a required tool?